pallets/ajuna-awesome-avatars/src/tests.rs

```diff
@@ -615,6 +615,61 @@ mod season {
 			);
 		});
 	}
+
+	#[test]
+	fn active_season_hooks_should_update_active_season_on_block_initialize() {
```

Comment:
For happy paths, let's stick to `should_work` for consistency
```suggestion
	fn active_season_hooks_should_work() {
```